src/locales/index.js

```diff
@@ -5,6 +5,7 @@ import es from './es.json'
 import enMobile from './mobilePhrases/en.json'
 import esMobile from './mobilePhrases/es.json'
 import { isDesktop, parseTags } from '../components/utils'
+export { localised, LocaleProvider } from './context'
```

Comment:
Do you need `LocaleProvider` here? It seems you aren't using it anywhere in this file

Reply:
It's `export`ing rather than `import`ing 😛

Reply:
Oops! I'm so used to see imports at the top and exports at the bottom so I didn't realise :)